### ISSUE TITLE
Question about the grid_to_lidar_unproject function

### ISSUE THREAD

**Issue by taylover-pei** [TRAILab/CaDDN#61]
I have read your paper and it is really a great job.

Since I'm a new one in this field, I have a question about the code.

In the below code, what does the ' Calculate grid to LiDAR unprojection for each plane' actually means? 
In my opinion, x_size/y_size/z_size means one point in the grid represents x_size meters in the lidar coordinates, is that right? 
So what's the relationship between x_size/y_size/z_size and the unproject tensor?
https://github.com/TRAILab/CaDDN/blob/5a96b37f16b3c29dd2509507b1cdfdff5d53c558/pcdet/models/backbones_3d/f2v/frustum_grid_generator.py#L45

Thank you very much. Looking forward to your reply!

**Comment by codyreading** [comment]
Yup thats correct, this transformation converts coordinates in the voxel grid (indices that range from 0 to the maximum shape of the grid) to real world coordinates (in the LiDAR frame in metres). The `unproject` tensor represents that transformation, as it scales multiplies the voxel coordinates by the voxel size and adds the origin of the coordinate frame (`pc_min`).
